rest-api-spec/src/main/resources/rest-api-spec/test/indices.rollover/35_max_single_primary_size_condition.yml

```diff
@@ -0,0 +1,58 @@
+---
+"Rollover with max_single_primary_size condition":
+  - skip:
+      version: " - 7.11.99"
```

Comment:
Double check my brain on this one bit here -- is this the right version?

Reply:
this will skip the test on 7.x, I think you want `7.10.99` instead

Reply:
Yes this is the right version